### PR TITLE
Audio output of detected notes

### PR DESCRIPTION
**Pull request by Juphex**
Closes https://github.com/UHHRobotics22-23/marimbabot/issues/39

### CODE REVIEW

**Comment by Juphex** [comment]
I need to test it yet

File: marimbabot_audio/src/audio_from_lilypond.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    lilypond_file  = create_lilypond_file(sentence)
+
+    # store lilypond as midi temporarily
+    # store temporarily due to parser limitations concerning '\midi' https://abjad.github.io/api/abjad/parsers/parser.html 
+    temp_ = tempfile.TemporaryFile()
+    as_midi(lilypond_file, str(temp_.name), remove_ly=True, resolution=200)
+    midi_filename = str(temp_.name) + ".mid"
+
+    # create audio from midi
+    # it is hardcoded for now, could be changed to a temporary file as well
```
Comment (Flova) [inline marimbabot_audio/src/audio_from_lilypond.py:45]: I think we should definitely make it a temp file

File: marimbabot_audio/src/audio_from_lilypond.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    sound_request.command = 1
+    sound_request.volume = 0.5
+    sound_request.arg = audio_filename
+
+    audio_publisher.publish(sound_request)
+
+# create audio from lilypond publisher
+# send audio to audio node (sound_play package)
+def listener():
+    rospy.init_node('visualization_node')
```
Comment (Flova) [inline marimbabot_audio/src/audio_from_lilypond.py:72]: The node name is prone to namespace issues. Maybe all audio to it.

**Review by Flova — CHANGES_REQUESTED** [review]

**Comment by Juphex** [comment]
the package midi2audio depends on fluidsynth https://github.com/ros/rosdistro/pull/37007

**Review by Flova — APPROVED** [review]

**Comment by Juphex** [comment]
I accidentally pushed some dev files... @Flova

**Comment by Juphex** [comment]
I resolved the merge conflicts with the main branch but I am not sure about all the imports that are within the main https://github.com/UHHRobotics22-23/marimbabot/pull/17#issuecomment-1537545534.

**Pull request merged**